flow/src/org/labkey/flow/controllers/well/showWell.jsp

```diff
@@ -305,7 +305,7 @@ function showKeywords()
     for (var i=0 ; i<keywords.length ; i++)
     {
         var o = keywords[i];
-        o.label = o.index == 0 ? 'Keywords' : 'Parameter ' + _pad(o.index) + ' -- ' + o.name;
+        o.label = o.index == 0 ? h('Keywords') : h('Parameter ' + _pad(o.index) + ' -- ' + o.name);

```

Comment:
This is JavaScript code in the middle of a JSP! h() is not going to work there. I don't think you need any changes here.